### ISSUE TITLE
Running a parallel LU test on the CI

### ISSUE THREAD

**Issue by amneetb** [IBAMR/IBAMR#1711]
I am using sparse direct solver for the acoustic flow solver. Locally I have built PETSc with MUMPS and I can run stuff in parallel. I'm not sure if PETSc on the CI is built with that option. Currently I have added a serial test in my branch for it
https://github.com/IBAMR/IBAMR/blob/amneetb/acoustically-driven-flows/tests/IBTK/acoustic_streaming_hier_integrator_2d.input#L413 

In the future I would like to add tests in parallel as well. Can we build PETSc with MUMPS on the CI (if not done already) in order for the test to pass?

**Comment by drwells** [comment]
No, I don't think that it is, but I want to better understand what you want to accomplish here before we change anything.

Do you plan on using this in more than just a single example (i.e., is this going to the library)? If so, then we should add it as an explicit optional dependency to IBAMR. If it's just a single example then it isn't really part of the library and doesn't need to be in the test suite.

**Comment by amneetb** [comment]
Yes, more than one example. The acoustic solver "requires" a direct solver because we don't have a preconditioner for the iterative solver yet (same in the literature).

**Comment by drwells** [comment]
Got it. It sounds like we should set this up as a proper optional dependency first.

**Comment by drwells** [comment]
Want to write a patch to set this up as a configuration option?
